shared/constants/teams.js

```diff
@@ -265,6 +265,12 @@ const hasChannelInfos = (state: TypedState, teamname: Types.Teamname): boolean =
 const getTeamMemberCount = (state: TypedState, teamname: Types.Teamname): number =>
   state.teams.getIn(['teammembercounts', teamname], 0)
 
+const isLastOwner = (state: TypedState, teamname: Types.Teamname): boolean => {
+  const allTeamMembers = state.teams.teamNameToMembers.getIn([teamname], I.Map())
+  const countOfOwners = allTeamMembers.count(m => m.type === 'owner')
```

Comment:
not a big deal but if you check your own role you can opt out of looking at this entire list of members

Reply:
done